zebrad/tests/common/get_block_template_rpcs.rs

```diff
@@ -0,0 +1,164 @@
+use super::*;
+
+pub(crate) mod submit_block {
```

Comment:
Nitpick: Can you create a separate module file, rather than using an inline module?

Inline modules are a bit harder to read, because all their code is indented.

Reply:
Absolutely. Updated in [`bc311c6`](https://github.com/ZcashFoundation/zebra/pull/5526/commits/bc311c62e3c1378d2d7a59751ee56a3a99db45a2)